x-pack/test/security_solution_endpoint/page_objects/ingest_manager_create_package_policy_page.ts

```diff
@@ -101,5 +103,38 @@ export function IngestManagerCreatePackagePolicy({
       });
       await this.ensureOnEditPageOrFail();
     },
+
+    /**
+     * Returns the Endpoint Callout that is displayed on the Integration Policy create/edit pages
+     */
+    async findEndpointActionsButton() {
+      const button = await testSubjects.find('endpointActions');
+      await this.scrollToCenterOfWindow(button);
+      return button;
+    },
+
+    /**
+     * Center a given Element on the Window viewport
+     * @param element
+     */
+    async scrollToCenterOfWindow(element: WebElementWrapper) {
```

Comment:
nice

Reply:
I'm still finding out if this would be a good contribution to the test framework `pageObjects`. If not, then I will likely move it to our common `pageObjects` for reuse